molecule/caddy/molecule.yml

```diff
@@ -20,6 +20,15 @@ platforms:
 provisioner:
   name: ansible
   inventory:
-    all: {}
+    group_vars:
+      all:
+        caddy_backends_healthcheck_port: 8081
```

Comment:
If our backend is running on port 8081, and it's down, we should get `503` status code instead of sending the traffic for that backend